api/models/logger.go

```diff
@@ -47,6 +56,13 @@ type LoggerConfig struct {
 	Mode    LoggerMode `json:"mode"`
 }
 
+func (lc *LoggerConfig) ToValidConfig() *LoggerConfig {
+	if !lc.Mode.isValid() {
+		lc.Mode = LogAll
```

Comment:
Should we add `lc.Enabled = false` too?

Reply:
I understand this function incorrectly. I think it's better to rename this function to `SetModeToDefault()` as it's more explanatory.

Reply:
it is not necessary set mode to default, it changed the mode to default one if the model is not valid. Does `SetModeToDefaultIfNecessary()` make sense?

Reply:
I think it's too long 😂 
I suggest `ToValidMode` or `SanitizeMode`.

Oh ya, any reason why is the function returning the *LoggerConfig? The receiver already a pointer so the method already changes the receiver value.

Reply:
no specific reason, just habit not create function that only mutate its value, I can make it returning nothing